financial-connections-example/src/main/java/com/stripe/android/financialconnections/example/FinancialConnectionsLauncherActivity.kt

```diff
@@ -1,89 +1,230 @@
+                subtitleResId = R.string.financial_connections_data_subtitle,
+                klass = FinancialConnectionsComposeExampleActivity::class.java,
+                section = MenuItem.Section.CompleteFlow,
+            ),
+
+            MenuItem(
+                titleResId = R.string.financial_connections_data_java_title,
+                subtitleResId = R.string.financial_connections_data_subtitle,
+                klass = FinancialConnectionsDataExampleActivityJava::class.java,
+                section = MenuItem.Section.CompleteFlow,
+            ),
+
+            MenuItem(
+                titleResId = R.string.financial_connections_payouts_title,
+                subtitleResId = R.string.financial_connections_payouts_subtitle,
+                klass = FinancialConnectionsBankAccountTokenExampleActivity::class.java,
+                section = MenuItem.Section.CompleteFlow,
+            ),
+
+            MenuItem(
+                titleResId = R.string.financial_connections_webview_title,
+                subtitleResId = R.string.financial_connections_webview_subtitle,
+                klass = FinancialConnectionsWebviewExampleActivity::class.java,
+                section = MenuItem.Section.Alternative,
+            ),
+
+            MenuItem(
+                titleResId = R.string.financial_connections_playground_title,
+                subtitleResId = R.string.financial_connections_playground_subtitle,
+                klass = FinancialConnectionsPlaygroundActivity::class.java,
+                section = MenuItem.Section.Internal,
+            ),
+        )
+    }
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
-        setContentView(R.layout.activity_financialconnections_launcher)
-
-        val linearLayoutManager = LinearLayoutManager(this)
-            .apply {
-                orientation = LinearLayoutManager.VERTICAL
+        setContent {
+            FinancialConnectionsExampleTheme {
+                MainScreen(items = items)
             }
+        }
+    }
 
-        findViewById<RecyclerView>(R.id.examples).run {
-            setHasFixedSize(true)
-            addItemDecoration(
-                DividerItemDecoration(
-                    this@FinancialConnectionsLauncherActivity,
-                    linearLayoutManager.orientation
-                )
+    @Composable
+    private fun MainScreen(items: List<MenuItem>) {
+        val groupedItems = remember(items) {
+            items.groupBy(MenuItem::section)
+        }
+
+        LazyColumn {
+            section(
+                title = "Recommended Integrations",
+                items = groupedItems.getOrElse(MenuItem.Section.CompleteFlow) { emptyList() },
+            )
+
+            section(
+                title = "Alternatives",
+                badge = { NotRecommended() },
+                items = groupedItems.getOrElse(MenuItem.Section.Alternative) { emptyList() },
```

Comment:
Should we collapse this section to discourage use, since it’s not the recommended approach?

Reply:
Good idea! Added expandable capabilities to sections and made the non recommended hidden by default: https://github.com/stripe/stripe-android/pull/6838/commits/d6507a234c3617edf509d6f1be07a3bcb28d1dbc

![image](https://github.com/stripe/stripe-android/assets/99293320/cd23290e-54c3-464d-934e-b8f72924cf8b)

Reply:
Nice! Might be a good idea to limit it to the alternatives. Can you do that?

Reply:
made collapsable optional - https://github.com/stripe/stripe-android/pull/6838/commits/423076c100b91250a2b2b512b3607613485df766

<img width="445" alt="image" src="https://github.com/stripe/stripe-android/assets/99293320/f91645d2-ec26-4087-82d5-b672e49b0757">